app/components/map.tsx

```diff
@@ -0,0 +1,93 @@
+"use client";
+
+import React, { useRef, useEffect } from "react";
+import mapboxgl, { LngLat } from "mapbox-gl";
+import "mapbox-gl/dist/mapbox-gl.css";
+
+const lookupCoordinates = {
```

Comment:
temporary and for reference; will be replaced at some point by props being passed in